docs/alma/alma.rst

```diff
@@ -127,7 +127,9 @@ You can query by object name or by circular region:
 .. doctest-remote-data::
 
     >>> from astroquery.alma import Alma
-    >>> m83_data = Alma.query_object('M83')
+    >>> alma = Alma()
+    >>> alma.archive_url = 'https://almascience.eso.org'  # optional to make doctest work
```

Comment:
I wonder whether we should hide this from the documentation e.g. setting it in the config that is being hidden from the rendering, so users are not copy pasting it?

 cc @keflavich?

Reply:
Yeah, this seems unnecessary to include in the documentation since users should default to the default auto-redirect tool?

But otoh, there should be an example of how to change servers because too often one or more are down